Server/mods/deathmatch/logic/luadefs/CLuaAccountDefs.cpp

```diff
@@ -70,6 +71,7 @@ void CLuaAccountDefs::AddClass(lua_State* luaVM)
 
     lua_classfunction(luaVM, "setData", "setAccountData");
     lua_classfunction(luaVM, "setPassword", "setAccountPassword");
+    lua_classfunction(luaVM, "setSerial", "SetAccountSerial");
```

Comment:
SetAccountSerial should start with a lowercase letter here

Reply:
Good point, I hadn't noticed 62ebf06